divisible.rb

```diff
@@ -0,0 +1,7 @@
+def divisible_by_2_3_5
+  result = []
+  (1..100).each { |i| result << i if i % 2 == 0 || i % 3 == 0 || i % 5 == 0 }
+  result
+end
+
+print divisible_by_2_3_5.join(', ')
```

Comment:
This looks good. I did see the other PR where you updated this to use `.filter`. This is an excellent use case for that method, well done on the refactor!

Reply:
Thanks